Tests/MongoSwiftTests/ConnectionStringTests.swift

```diff
@@ -203,6 +211,13 @@ final class ConnectionStringTests: MongoSwiftTestCase {
         try self.runTests("connection-string")
     }
 
+    func testCodable() throws {
+        let connStr = try MongoConnectionString(throwsIfInvalid: "mongodb://localhost:27017")
+        let encodedData = try JSONEncoder().encode(connStr)
+        let decodedResult = try JSONDecoder().decode(MongoConnectionString.self, from: encodedData)
+        expect(connStr.description).to(equal(decodedResult.description))
```

Comment:
we also want to add an additional assertion here to ensure the `.description` is working correctly and evaluates to `"mongodb://localhost:27017"`. For example, if `.description` just returned the empty string, this test would currently pass as written.

Reply:
Done